### PR TITLE
fix(analytics) : export vars for analytics snapshot

### PR DESCRIPTION
**Pull request by invincibleJai**
### Analytics Snapshot URL 

These variables are needed for snapshot URLs which is generated on PRs in fabric8-ui, in order to test Dependency editor (One new feature). 

export vars for analytics snapshot

### CODE REVIEW

File: vars/buildSnapshotFabric8UI.groovy [hunk 1]
```diff
@@ -29,6 +29,8 @@ def call(body) {
         export FABRIC8_FORGE_API_URL="https://forge.api.prod-preview.openshift.io"
         export FABRIC8_SSO_API_URL="https://sso.prod-preview.openshift.io/"
         export FABRIC8_AUTH_API_URL="https://auth.prod-preview.openshift.io/api/"
+        export ANALYTICS_RECOMMENDER_URL="https://recommender.api.prod-preview.openshift.io/"
```
Comment (hrishin) [inline vars/buildSnapshotFabric8UI.groovy]: @invincibleJai Are we sure about `prod-preview` links?

Comment (jaseemabid) [inline vars/buildSnapshotFabric8UI.groovy]: There is a FABRIC8_RECOMMENDER_API_URL already, will that do?

Comment (invincibleJai) [reply vars/buildSnapshotFabric8UI.groovy]: Am not sure @jaseemabid what is that used for as it was already there , if it's unused we need to clean it up.

We need `ANALYTICS_RECOMMENDER_URL ` as it's there in the template and same has been updated in prod/ prod-preview env https://github.com/fabric8-ui/fabric8-ui/blob/master/openshift/fabric8-ui.app.yaml#L47

Comment (invincibleJai) [reply vars/buildSnapshotFabric8UI.groovy]: And regarding another thing, we discussed i.e prefixing these vars with FABRIC8 will require changes in fabric8-ui and need to have HK issue to update those https://github.com/fabric8-ui/fabric8-ui/blob/master/openshift/fabric8-ui.app.yaml#L47 , so will go ahead and make those changes if it's really needed. However changes made in this PR will help to test badger and be sure if things are working as expected.

**Comment by invincibleJai** [comment]
Closing it , will reopen if needed

**Pull request closed without merging**